### PR TITLE
Updating USNG4J to match PR changes to USNG.js

### PR DESCRIPTION
**Pull request by BernardIgiri**
Correcting northing value conversions for UTM conversions and adding the isInUPSSpace helper method.

### CODE REVIEW

**Comment by BernardIgiri** [comment]
USNG.js PR: https://github.com/codice/usng.js/pull/26

File: usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java [hunk 1]
```diff
@@ -288,23 +288,18 @@ public UtmCoordinate toUtm(final DecimalDegreesCoordinate decimalDegreesCoordina
...
+    UTMNorthing += (UTMNorthing < 0) ? NORTHING_OFFSET : 0;
 
     return new UtmCoordinateImpl(zoneNumber, UTMEasting, UTMNorthing);
   }
 
   @Override
   public UpsCoordinate toUps(final DecimalDegreesCoordinate decimalDegreesCoordinate) {
-    final UtmUpsCoordinate utmUpsCoordinate = toUtmUps(decimalDegreesCoordinate);
-    if (utmUpsCoordinate.isUTM()) {
+    if (!isInUPSSpace(decimalDegreesCoordinate)) {
```
Comment (dimitry-sherman) [inline usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java:298]: ❓ Why was this put in instead of `isUTM`?

Comment (BernardIgiri) [reply usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java:298]: isUTM is based on the utmUpsCoordinate and is only available after conversion from lat lon, isInUPSSpace checks decimalDegreesCoordinates which is a lat lon coordinate.

Comment (Lambeaux) [reply usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java:298]: ❓ Isn't there some overlap between UTM and UPS space?

File: usng4j-api/src/main/java/org/codice/usng4j/CoordinateSystemTranslator.java [hunk 2]
```diff
@@ -167,4 +167,10 @@ UsngCoordinate toUsng(
    * @throws ParseException when 'msgrsString' isn't in MGRS format.
    */
   UsngCoordinate parseMgrsString(String mgrsString) throws ParseException;
+
+  /**
+   * @param decimalDegreesCoordinate longitude latitude coordinate
+   * @return true if latitude is in UPS space
+   */
+  boolean isInUPSSpace(DecimalDegreesCoordinate decimalDegreesCoordinate);
```
Comment (Lambeaux) [inline usng4j-api/src/main/java/org/codice/usng4j/CoordinateSystemTranslator.java:175]: ❓ Do we already have a method stub for this? 
https://github.com/codice/usng4j/blob/master/usng4j-api/src/main/java/org/codice/usng4j/UtmUpsCoordinate.java#L40

File: usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java (hunk 1 above)
Comment (BernardIgiri) [reply usng4j-impl/src/main/java/org/codice/usng4j/impl/CoordinateSystemTranslatorImpl.java:298]: No.

File: usng4j-api/src/main/java/org/codice/usng4j/CoordinateSystemTranslator.java (hunk 2 above)
Comment (BernardIgiri) [reply usng4j-api/src/main/java/org/codice/usng4j/CoordinateSystemTranslator.java:175]: That's in UtmUpsCoordinate which is only available after a unit conversion. The lat/lon coordinate is what should be used to determine that which is available before conversion.

**Pull request closed without merging**